dist/osx/README.debugging

```diff
@@ -0,0 +1,28 @@
+Arduino sketches can be debugged with GDB using an OpenOCD session over a JTAG
+connection.
+
+After uploading a sketch to the board, run these commands in SEPARATE windows
+1. "<openocd_dir>/bin/openocd " 
+      -f "<openocd_dir>/scripts/interface/ftdi/flyswatter2.cfg" 
+      -f "<openocd_dir>/scripts/board/quark_se.cfg"
+2. <gdb> "<tmp_dir>/arduino101_sketch.debug.elf"
+
+In GDB, run the command: 
+   target remote localhost:3333
```

Comment:
Can you please tell me the expected output of this command? I got this far but I'm not sure what to do afterwards.